js/tests/Test_jsoo.ml

```diff
@@ -23,10 +23,13 @@ open Semgrep_js_shared
 (* Code *)
 (*****************************************************************************)
 
+let filtered = [ "Cpp"; "C++"; "Julia"; "Ruby" ]
```

Comment:
can we add a comment re: why we're skipping these?

Reply:
Yup